pages/[address].tsx

```diff
@@ -266,12 +269,14 @@ const SynthPage: React.FC<Props> = ({ data, relatedSynths }) => {
       refetchLongTokenBalance();
       refetchShortTokenBalance();
```

Comment:
just out of curiousity, would you need to do the same thing for long/short balances, or are those taken care of?

Reply:
Because the API changes the values used in the hooks above, I think this is ok.

Reply:
took a look at these calls and it looks like they are updated correctly